people/v_people.staff_roster.sql

```diff
@@ -338,7 +355,10 @@ SELECT c.employee_number
       ,c.file_number
       ,c.first_name
       ,c.last_name
-      ,c.position_status
+      ,CASE 
+        WHEN c.position_status = 'Deceased' THEN 'Terminated'
```

Comment:
this might need to happen more upstream, there are other CASE statements that look at position_status